packages/grid/src/features/standard-table/hooks/UseLocalStateTableContext.ts

```diff
@@ -0,0 +1,40 @@
+import { useMemo, useReducer } from "react";
+import { TableContext } from "../context/StandardTableStateContext";
+import { createStandardTableActions } from "../redux/StandardTableActionsAndSelectors";
+import {
+  createStandardTableReducer,
+  standardTableInitialState,
+  StandardTableReducer
+} from "../redux/StandardTableReducer";
+
+export const useLocalStateTableContext = <TColumnKeys extends string>(
+  tableId: string
+) => {
+  const reducer = useMemo(
```

Comment:
no `useMemo` neccessary here, just inline it in `useReducer`

Reply:
That would recreate the reducer tree on every render.

Reply:
Removed memoize.